### DISCUSSION TITLE
Question: Index?

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#61] [Week 2 - Questions]
Is your index.html file where all of your information is stored? Is it auto-created when you start your code or will you have to create it?

**Comment by audreyhki** [comment]
Just beginning on Week 2 Assignments and want to ask fellow classmates if they used the exercism editor or downloaded the CLI?

**Reply by metasean** [reply]
I recommend the the exercism editor.

**Reply by audreyhki** [reply]
Great! That's what I chose.

**Comment by metasean** [comment]
@CJ3315 - For your project, you'll have to create all the files, including the `index.html` file.  

Before you need to do that, I'll provide some additional instructions.  When we get there, I'll provide a short video for getting started on your actual projects.  (In addition to creating specific files, you'll also need an introduction to the Integrated Development Environment (IDE), & how to serve your pages.  More importantly, you shouldn't need to know any of this until you start your project, which you shouldn't be doing for another few weeks.)
